packages/shared/lib/contexts/onboarding/actions/cleanupProtectionOnboarding.ts

```diff
@@ -14,7 +16,7 @@ export async function cleanupProtectionOnboarding(pinInput: string): Promise<voi
     await Platform.PincodeManager.set(get(newProfile)?.id, pinInput)
     // TODO: replace with new api when it is implemented
     // await setStoragePassword(pinInput)
-    if (get(walletSetupType) === SetupType.Mnemonic) {
-        await storeAndCleanMnemonic()
-    }
+    // if (get(profileRecoveryType) === ProfileRecoveryType.Mnemonic) {
+    //     await storeAndCleanMnemonic()
+    // }
```

Comment:
Initially it didn't seem like something that should've been in this function, but I am not sure really sure why I thought this because it seems like a good place for it :sweat_smile: